app/models/concerns/auditable_model.rb

```diff
@@ -7,13 +7,23 @@ module AuditableModel
     private
 
     def audit_created!(activity_type:, activity_information: nil, audit_comment: nil)
-      audits.create!(
-        user: Current.user,
-        activity_type: activity_type,
-        activity_information: activity_information,
-        audit_comment: audit_comment,
-        api_user: Current.api_user
-      )
+      if is_a?(PlanningApplication)
+        audits.create!(
+          user: Current.user,
+          activity_type: activity_type,
+          activity_information: activity_information,
+          audit_comment: audit_comment,
+          api_user: Current.api_user
+        )
+      else
+        planning_application.audits.create!(
+          user: Current.user,
+          activity_type: activity_type,
+          activity_information: activity_information,
+          audit_comment: audit_comment,
+          api_user: Current.api_user
+        )
+      end
```

Comment:
How about an `audits` method in `document.rb` and `validation_request.rb`?

```ruby
def audits
  planning_application.audits
end
  ```
  
Then this can just be:

```ruby
def audit_created!(activity_type:, activity_information: nil, audit_comment: nil)
   audits.create!(
      user: Current.user,
      activity_type: activity_type,
      activity_information: activity_information,
      audit_comment: audit_comment,
      api_user: Current.api_user
   )
end
  ```

So no conditional would be needed here

Reply:
Or even better i think you can just use a delegator
i.e. 
```ruby
delegate :audits, to: :planning_application
```